src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -1,43 +1,38 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
 
+import { colorLightGrey } from '../../styles/designTokens';
+import { formatDate } from '../../utils';
 import Shipping from '../Icons/Shipping';
```

Comment:
Can you import Icon this way? It will be easier to read in `render` method.
`import { Shipping as ShippingIcon } from '../Icons';`

Also do not import icons by default export directly from Icon file. We have `index.js` file inside `Icons` folder, that you should use for importing.